docs/docs/model-storage.mdx

```diff
@@ -2,156 +2,214 @@
 id: model-storage
 sidebar_label: Model Storage
 title: Model Storage
+abstract: |
+  After you've trained a model for your assistant, you can use it to run your
+  bot.
 ---
 
-Rasa can load your model in three different ways:
+You can load your trained model
+in three different ways:
 
-1. Fetch the model from a server (see [Fetching Models from a Server](./model-storage.mdx#server-fetch-from-server)), or
+1. Load the model from your local disk (see [Load Model from Disk](./model-storage.mdx#load-model-from-disk))
 
-2. Fetch the model from a remote storage (see [Remote Storage](./model-storage.mdx#server-fetch-from-remote-storage)).
+2. Fetch the model from your own http server (see [Load Model from Server](./model-storage.mdx#load-model-from-server))
 
-3. Load the model specified via `-m` from your local storage system.
+3. Fetch the model from cloud storage like S3 (see [Load Model from Cloud](./model-storage.mdx#load-model-from-cloud))
 
-Rasa tries to load a model in the above mentioned order, i.e. it only tries to load your model from your local
-storage system if no model server and no remote storage were configured.
+Loading the model from your local disk will be used as a fallback if you
+don't specify how a model should be loaded.
```

Comment:
Can we rephrase this to make it sound more like a default than a fallback (this here feels like it implies you need to specify)

Reply:
the intro to the next section is much clearer about this!